<?xml version="1.0"?>
<launch>
    <!-- spawn Upper controller  !! -->
    <node pkg="asoc_upper_controller" type="upper_controller_angle" name="upper_controller" clear_params="true" output="screen">
        <param name = "controller_freq" type="double" value = "40.0"/>
        <param name = "goalRadius" type="double" value = "0.5"/>
        <param name = "goal_pose_err" type="double" value = "1.0"/>

        <param name = "baseSpeed" type="double" value = "1.5"/>
        <param name = "rot_angle" type="double" value = "45.0"/>
        <param name = "P_Long" type="double" value = "30.0"/>
        <param name = "I_Long" type="double" value = "0.0"/>
        <param name = "D_Long" type="double" value = "2.0"/>

        <param name = "P_Yaw" type="double" value = "60.0"/>
        <param name = "I_Yaw" type="double" value = "0.0"/>
        <param name = "D_Yaw" type="double" value = "0.1"/>
        <param name = "forward_dist" type="double" value = "1.3"/>
        

        <param name = "P_Lateral" type="double" value = "45.0"/>
        <param name = "I_Lateral" type="double" value = "0.0"/>
        <param name = "D_Lateral" type="double" value = "3.0"/>


    </node>
</launch>

